demisto_sdk/commands/common/docker_helper.py

```diff
@@ -50,6 +57,7 @@ def init_global_docker_client(timeout: int = 60, log_prompt: str = ""):
         docker_user = os.getenv("DOCKERHUB_USER")
         docker_pass = os.getenv("DOCKERHUB_PASSWORD")
         if docker_user and docker_pass:
+            logger.info(f"{log_prompt} - logging in to docker registry")
```

Comment:
What's the point of `log_prompt`?

Reply:
This is currently used (not new behavior). the log prompt usually contain the integration name